src/core/shape_inference/include/irdft_shape_inference.hpp

```diff
@@ -26,11 +26,11 @@ std::vector<TRShape> shape_infer(const IRDFT* op,
     auto axes = get_input_const_data_as<TRShape, int64_t>(op, 1, ta);
     auto axes_are_known = static_cast<bool>(axes);
 
-    util::rfft_common_validation::shape_validation(op,
-                                                   input_shapes,
-                                                   *axes,
-                                                   axes_are_known,
-                                                   util::rfft_common_validation::RFFTKind::Inverse);
+    util::fft_common_validation::shape_validation(op,
+                                                  input_shapes,
+                                                  *axes,
+                                                  axes_are_known,
```

Comment:
As part 2 (improvments)?: The axes_are_known paramter I think can be removed by wrap this call in if (axes != nullptr) or do this check inside function.